src/driver.py

```diff
@@ -0,0 +1,49 @@
+import os
+from argparse import ArgumentParser
+from spec_parser import SpecParser
+
+
+def get_args():
+
+    argparser = ArgumentParser(prog='spec-parser',
+                               description='SPDX specification parser')
+
+    argparser.add_argument('model', type=str,
```

Comment:
One argument, the top-level directory.

Reply:
Resolved! now it expects directory `spec-dir` containing all namespaces..